### PR TITLE
chore: Added simple / inline api setup

### PR DESCRIPTION
**Pull request by vinnyjth**
## DESCRIPTION

@jimmyhogoboom if you test this, don't spend too much time on it if it gives you problems. This is a quick stab on a way to do API development in only one repo. Follow the instructions in the readme.md and let me know what you think.

### CODE REVIEW

**Comment by redreceipt** [comment]
I do like this idea a lot. correct me if I'm wrong, taking this philosophy to its end, we could eliminate the need for linking all together in the templates repo?

**Comment by vinnyjth** [comment]
If we also do this for the App, yes!

**Comment by vinnyjth** [comment]
This has some unique differences to linking, if you make any changes to the Api/App they are lost by default. You need to copy them over to `templates`, or make them in `templates` first. I think that's okay as long as linking is technically possible (even if not the default)

**Comment by redreceipt** [comment]
I think if we do this, we remove linking completely. the advantage of this in my mind is the infrastructure simplicity it brings to the templates repo. 

I don't think it's a bad thing that if there's specific template work that needs to be done, you do it there first with a PR, then work on your apps PR with the updated template.

**Comment by redreceipt** [comment]
said another way, if the only reason we do this is to make our lives easier on development, I don't think it's worth it

**Comment by vinnyjth** [comment]
👍 I don't 100% agree, but I tend to be more pessimistic with our future needs. I'm fine ditching linking and giving just this strategy a shot.

**Comment by redreceipt** [comment]
we can slow play it. leave our templates repo unlinked and pretend as if we don't have it and try it for a month

File: scripts/setup-api.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+shopt -s dotglob
+mv apollos-templates/apollos-church-api/* ../ApollosApi
+
+
+
+cd ..
+
+node scripts/swap-package-json-to-links.js ./ApollosApi
+
+rm -rf tmp
```
Comment (redreceipt) [inline scripts/setup-api.sh:35]: this bash script makes my heart happy

File: scripts/setup-api.sh [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+mkdir -p tmp
+
+echo $2
+BRANCH=$2
+
+cd tmp
+git clone git@github.com:ApollosProject/apollos-templates.git
+git checkout ${BRANCH:-master}
+
+cd apollos-templates
```
Comment (redreceipt) [inline scripts/setup-api.sh:14]:
```suggestion
cd apollos-templates
git checkout ${BRANCH:-master}
```

File: scripts/setup-api.sh [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+mkdir -p ApollosApi
+mkdir -p tmp
+
+echo $2
+BRANCH=$2
+
+cd tmp
+git clone git@github.com:ApollosProject/apollos-templates.git
+git checkout ${BRANCH:-master}
```
Comment (redreceipt) [inline scripts/setup-api.sh:12]:
```suggestion
```

File: scripts/setup-api.sh (hunk 2 above)
Comment (vinnyjth) [reply scripts/setup-api.sh:13]: good catch

**Review by redreceipt — APPROVED** [review]

**Pull request merged**